packages/aws-cdk-lib/README.md

```diff
@@ -0,0 +1,51 @@
+# AWS Cloud Development Kit Library
+
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+
+The AWS CDK construct library provides APIs to define your CDK application and add
+CDK constructs to the application.
+
+## Usage
+
+### Upgrade from CDK 1.x
+
+When upgrading from CDK 1.x, remove all dependencies to individual CDK packages
+from your dependencies file and follow the rest of the sections.
+
+### Installation
+
+Add a single entry in your dependencies list to this package.
+
+You also need to add a reference to the `constructs` library, according to the
+kind of project you are developing:
+- For libraries, model the dependency under `devDependencies` **and** `peerDependencies`
+- For apps, model the dependency under `dependencies` only
+
+### Use in your code
+
+#### Classic import
```

Comment:
What about the [rewrite imports script](https://www.npmjs.com/package/@monocdk-experiment/rewrite-imports)?

Reply:
I have a separate line item in my plan for this package. This will be addressed as part of that. Is that ok if we update this after we decide how the rewrite import script will be packaged in v2?

Reply:
make sense.